cli/CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 <!-- See the ../guides/writing-the-cypress-changelog.md for details on writing the changelog. -->
 ## 13.6.4
 
+_Released 1/30/2024 (PENDING)_
+
+** Dependency Updates:**
+
+- Updated  `@cypress/unique-selector` to include a performance optimization. It's possible this could improve performance of the selector playground. Addressed in [#28571](https://github.com/cypress-io/cypress/pull/28571).
+- Upgraded `electron` from `25.8.4` to `27.1.3`
+- Upgraded bundled Node.js version from `18.15.0` to `18.17.0`
+- Upgraded bundled Chromium version from `114.0.5735.289` to `116.0.5845.228`
```

Comment:
Chromium version is `v18.17` --  https://www.electronjs.org/docs/latest/tutorial/electron-timelines / showing in System Tests output as well